### DISCUSSION TITLE
Не работает touch scroll у элементов внутри ModalPage

### DISCUSSION THREAD

**Discussion by hhrum** [VKCOM/VKUI#6570] [Q&A]
При использовании любого скролящегося элемента(HorizontalScroll, Select) внутри ModalPage скролл не работает.

В результате некоторых копаний выяснилось, что событие onScroll просто не вызывается на дочерних компонентах у ModalPage. Скорее всего такое поведение происходит из-за компонента ModalRoot. А если быть точнее, то из-за обработчика onPageTouchMove, который вызывает preventDefault() для события touchmove

UPD: Не знал, куда засунуть это, но такое поведение доставляет большие не удобства с:

**Comment by SevereCloud** [comment]
Привет, эту проблему можно отследить здесь #335
